dev-docs/show-native-ads.md

```diff
@@ -166,19 +168,29 @@ And the following optional fields:
 + icon
 + cta
 
-A native `image` ad unit can be set up in the manner below:
-
-{% highlight js %}
+A native "image-type" ad unit can be set up as shown in the following example.
```

Comment:
Are you sure that this is the correct way to set up a native "image-type" ad unit?  This looks like standard native with just a single required asset.

Reply:
http://www.nooooooooooooooo.com x 2

Was writing it out longhand by mistake.  Fixed in b31cbb7 to use the shorthand.